internal/storage/local/local.go

```diff
@@ -0,0 +1,130 @@
+package local
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"time"
+
+	strg "github.com/offen/docker-volume-backup/internal/storage"
+	t "github.com/offen/docker-volume-backup/internal/types"
+	u "github.com/offen/docker-volume-backup/internal/utilities"
+	"github.com/sirupsen/logrus"
+)
+
+type LocalStorage struct {
+	*strg.StorageBackend
+}
+
+// Specific init procedure for the local storage provider.
```

Comment:
Godoc comments usually start with the very name of the function, else `go vet` will complain and I think it makes sense to adhere to this. I.e. such a comment would become

```
InitLocal creates a new StorageBackend that stores archives locally
```